frontend/src/components/ui/LightningCard.tsx

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { DateTime } from 'luxon';
+import StarIcon from '../icons/star';
+
+import './LightningCard.scss';
+import { ITalk } from '../../store/modules/talks/talks.types';
+import { useAppDispatch, useAppSelector } from '../../store';
+import { vote } from '../../store/modules/talks/talks.api';
+import UserIcon from '../icons/user';
+import ClockIcon from '../icons/clock';
+import { toast } from './ToastManager';
+import Spinner from './Spinner';
+
+type Props = ITalk;
+
+type State = {
```

Comment:
What is this for?

Reply:
Oh, I initially started using all class components, but then I changed some of them to functional to try/use hooks. I forgot to remove the State type from when I changed it over.

I'll remove that right away and check for any other dangling references from the change over.